src/utils/axiosSetup.test.ts

```diff
@@ -35,7 +35,7 @@ const mockInterceptorParams: InterceptorParams = {
 
 describe('axiosSetup', () => {
   test('interceptor calls onTokenRefreshFailed', async (done) => {
-    const mockErrorUnauthenticated: AxiosError = {
```

Comment:
pipeline is failing on this file (we had same issue last week), merge master in and try to format again.